internal/api.go

```diff
@@ -494,6 +494,9 @@ func Call(ctx netcontext.Context, service, method string, in, out proto.Message)
 	if ticket == "" {
 		ticket = DefaultTicket()
 	}
+	if dri := c.req.Header.Get("X-Appengine-Dev-Request-Id"); IsDevAppServer() && dri != "" {
```

Comment:
Define a var using `http.CanonicalHeaderKey`, as is done for the other headers?

Reply:
totally right, this was sloppy of me.